pkg/operator/util/admissionregistration.go

```diff
@@ -1,14 +1,14 @@
 package util
 
 import (
-	admregv1 "k8s.io/api/admissionregistration/v1beta1"
+	admregv1 "k8s.io/api/admissionregistration/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 )
 
-// ReadValidatingWebhookConfigurationV1Beta1OrDie reads a ValidatingWebhookConfiguration,
+// ReadValidatingWebhookConfigurationV1OrDie reads a ValidatingWebhookConfiguration,
 // as this is not yet added to library-go.
-func ReadValidatingWebhookConfigurationV1Beta1OrDie(objBytes []byte, scheme *runtime.Scheme) *admregv1.ValidatingWebhookConfiguration {
+func ReadValidatingWebhookConfigurationV1OrDie(objBytes []byte, scheme *runtime.Scheme) *admregv1.ValidatingWebhookConfiguration {
```

Comment:
Are there no users of the ReadMutatingWebhookConfigurationV1Beta1OrDie()? Can we just remove it?

Reply:
I renamed the func and here and its (one) [usage](https://github.com/openshift/hive/pull/1494/files#diff-b90095902244c03464a07cf8fd6ab580c5d445534cc97b1c750b2382792decebR143).

Reply:
I was asking about ReadMUTATINGWebhookConfigurationV1OrDie() (not Validating). You didn't touch it in the PR, but it the only other function in this file.

Reply:
Oh, my bad.

How about I remove it in a [separate PR](https://github.com/openshift/hive/pull/1508)?